templates/shared/_blender-contact-us-form.html

```diff
@@ -0,0 +1,78 @@
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>{{h1}}</h1>
+      <p>{{intro_text}}</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_{{formid}}">
+
+        <fieldset class="u-no-margin--bottom">
+          <h3>Tell us about your project and what problem(s) you’re looking to solve.</h3>
+          <input required id="Company" name="Company" maxlength="255" type="text" class="mktoField mktoRequired" />
+        </fieldset>
+
+        <fieldset class="u-no-margin--bottom">
+          <h3>How should we get in touch?</h3>
```

Comment:
A heading level is being skipped in the page heirarchy, at the moment it goes `h1` > `h3`, skipping `h2` - I'd question whether this even needs to be a heading element, it could be a `p` tag, and again, if it needs to be that font size, apply `.p-heading--three`.